apps/web/src/components/AccountSelectorModal/RemoveAccountModal.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { useImplicitAccounts, useRemoveAccount } from "@umami/state";
+
+import { RemoveAccountModal } from "./RemoveAccountModal";
+import {
+  act,
+  dynamicDisclosureContextMock,
+  render,
+  screen,
+  userEvent,
+  waitFor,
+} from "../../testUtils";
+
+jest.mock("@umami/state", () => ({
+  ...jest.requireActual("@umami/state"),
+  useRemoveAccount: jest.fn(),
+  useImplicitAccounts: jest.fn(),
+}));
+
+describe("<RemoveAccountModal />", () => {
+  const mockRemoveAccount = jest.fn();
+
+  beforeEach(() => {
+    (useRemoveAccount as jest.Mock).mockReturnValue(mockRemoveAccount);
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("renders with default description and button label", async () => {
+    (useImplicitAccounts as jest.Mock).mockReturnValue([{}, {}]);
+
+    render(<RemoveAccountModal account={{} as any} />);
```

Comment:
and if you do prepare the state, you wouldn't need to pass this meaningless value here